src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaExtensions.cs

```diff
@@ -50,6 +50,11 @@ public static bool IsMultiLineComment(this SyntaxTrivia trivia)
             return trivia.Kind() == SyntaxKind.MultiLineCommentTrivia;
         }
 
+        public static bool IsShebangComment(this SyntaxTrivia trivia)
+        {
+            return trivia.Kind() == SyntaxKind.ShebangCommentTrivia;
```

Comment:
Based on our API naming conventions, this will should be HashExclamationTrivia. Who from the scripting team is on the hook to follow the API naming guidelines? @tmat? @amcasey? @cston?